packages/xstate-fsm/test/fsm.test.ts

```diff
@@ -38,65 +37,65 @@ describe('@xstate/fsm', () => {
   it('should have the correct initial state', () => {
     const { initialState } = lightFSM;
 
-    assert.deepEqual(initialState.value, 'green');
-    assert.deepEqual(initialState.actions, [{ type: 'enterGreen' }]);
+    expect(initialState.value).toEqual('green');
```

Comment:
Before we merge this, I strongly prefer `assert` function syntax to `expect` syntax.

Reply:
I can change this back - that's not a problem. I'm wondering if it would be good though. 

Possible drawbacks:
- lack of cohesive docs (for some things u have to look into jest, for some u have look into i.e. chai
- possibly jest error messages are designed to understand their expect API better (and thus such errors can be more readable), i.e. they provide extra hints for times when a developer uses `toBe` instead of `toEqual` by accident
- some built-in features would be inaccessible (or would lead to 2 different styles being used in the codebase which is rather ugly), such as **inline** snapshots - which are useful i.e. for expecting error messages

Reply:
Okay, that's fair.